### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by e-v-medvedev**
update translation

### CODE REVIEW

**Comment by PierreRambaud** [comment]
Is it possible to write the same text in english (at least for document types with or without typo errors) because I can't translate it in english or french :/

**Comment by e-v-medvedev** [comment]
i`ll try :-). But i added some content only in document-type.md (from list above). In other files just translation or error fixing.

**Comment by PierreRambaud** [comment]
So please add english translation only for document type file ;)

File: ru_RU/reference-guide/datatypes.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-In this case, Datatype needs three classes to works:
+Класс "Тип данных" (Datatype) требует наличия следующих трех классов:
 
 * Datatype as Bootstrap
 * PrevalueEditor as Configuration
 * Editor as Content part
 
-Replace all texts in this **<span style="color:#FF9900;">color</span>** with the name you want to use for your datatype and classes must be added in "library/Datatypes/**<span style="color:#FF9900;">DatatypeName</span>**" directory.
+Replace all texts in this **<span style="color:#FF9900;">color</span>** with the name you want to use for your datatype and classes must be added in "library/Datatypes/**<span style="color:#FF9900;">DatatypeName</span>**" directory. (I think it will be good to explain where is this code placed. More information required.).
```
Comment (PierreRambaud) [inline ru_RU/reference-guide/datatypes.md]: Can you remove the note please?

**Comment by PierreRambaud** [comment]
Thanks ;)

**Pull request merged**